namespace-overview/design.md

```diff
@@ -1,21 +1,18 @@
 # Namespace Overview
 
 ## Summary
-The namespace overview would give a high-level summary of the health of each namespace (or composite) as well as some other basic information.
+The namespace overview would give a high-level summary of the health of the services in each namespace.
 
-## Card View
 ![card view of namespaces](img/namespace-cards.png)
-- Namespaces could be shown with their name and a rollup of their overall health as well as any of several different metrics
 
-## List View
-![list view of namespaces](img/namespace-list.png)
+### Namespace Cards
+- **Title:** Displays the name of the namespace. May wrap to a second line if necessary.
+- **Service Count:** The total number of services in the namespace.
+- **Status Counts:** Shows the number of services with each status. Icons should only be shown for a given status if there are services in the namespace that have that status.
 
-## Alternative Namespace Representations
-![alternative representations of namespaces](img/namespace-alternates.png)
-- Stats we could include
-  - Total number of services
-  - Component pieces of namespace health
-  - Summary of service status
-  - Traffic information
-  - Stats about services with ingresses
-  
+
+### Card interactions
+- Clicking on a card links to a view of the services list filtered by the relevant namespace
+- Clicking on a status count links to a view of the services with that status in the selected namespace
+- Hovering over a status count displays a popover with a list of the names of services with that status.
```

Comment:
If it's a hover, should it be a tooltip rather than an Infotip?  I don't remember if there is relevant PF guidance.

Reply:
yeah, tooltip would be a better word to use here. I'll update